com.unity.render-pipelines.universal/Runtime/2D/Light2D.cs

```diff
@@ -226,6 +221,27 @@ private void OnDisable()
             Light2DManager.DeregisterLight(this);
         }
 
+        internal static int CalculateHash(Light2D light)
```

Comment:
why not private? why static? why not `override GetHashCode()`

Reply:
Good idea. Will override hashCode..

Reply:
Fixed in f3026543226d53b04e4f15ae8f290d93917c92fc